src/twisted/web/newsfragments/10088.bugfix

```diff
@@ -0,0 +1 @@
+added support for `none` value for cookie's `samesite` attribute. Now it can be set like: `request.addCookie(samesite="none", secure=True)`. Only `secure` cookie mave have `none` value (check for reference: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#samesitesamesite-value)
```

Comment:
I think that for the release notes, we can keep this simple 

```suggestion
twisted.web.http.Request.addCookie now supports the `none` value for cookie's `samesite` attribute.
```

the devs using it should know that secure is required...and we have an expection if used without secure.